docs/spec-conformance.md

```diff
@@ -8,7 +8,6 @@ The following features are not implemented yet:
 Spec version | Feature                                  | PR
 -------------|------------------------------------------|----------------------------------------------------------
 v1.1.0       | `SECCOMP_FILTER_FLAG_WAIT_KILLABLE_RECV` | [#3862](https://github.com/opencontainers/runc/pull/3862)
-v1.1.0       | rsvd hugetlb cgroup                      | TODO ([#3859](https://github.com/opencontainers/runc/issues/3859))
 v1.1.0       | `.process.ioPriority`                    | [#3783](https://github.com/opencontainers/runc/pull/3783)
 
 
```

Comment:
The following lines about idmap are also outdated. Can you remove them too? Or shall I do another PR for that?

Reply:
Ah, yes, thanks! I'll remove these lines too.